### DISCUSSION TITLE
Can bitwuzla run in parallel?

### DISCUSSION THREAD

**Discussion by yflich** [bitwuzla/bitwuzla#126] [Q&A] [answered]
Is there any option to let the bitwuzla run in parallel, like z3 does?

**Comment by mpreiner** [comment, marked as answer]
Not yet, we plan to add support for parallel SAT solvers in the future.

**Comment by aniemetz** [comment]
@yflich Multi-threading support is now added on main. Use option `-j/--nthreads` to configure. Note that this only configures multi-threading in CryptoMiniSat (as the back end SAT solver) at the moment.

**Reply by yflich** [reply]
Thanks a lot
